ci/requirements-py35.yml

```diff
@@ -11,6 +11,7 @@ dependencies:
   - pandas
   - seaborn
   - scipy
+  - rasterio
```

Comment:
Rasterio supports 2.7 and 3.3-3.5 (https://github.com/mapbox/rasterio)

I added this requirement to all other test suites, was that correct? Version 1.0 is needed as many function names changed in rasterio at V1.0.